### PR TITLE
feat: made reset button more clear of its purpose

### LINKED ISSUE #100: [REQUEST] Reset Button Function unclear
**Is your feature request related to a problem? Please describe.**

**Describe the feature you'd like to see implemented**

Maybe the reset button should be described further to avoid any issues being report even tho there might be none. Or maybe even hide the button until its function is needed by a user

**Additional context**

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by ZigTag**
Fixes #100 

## Summary of Changes

Made reset button more clear of its actions

## Screenshots (if necessary)

Before:
image
After:
image

### CODE REVIEW

File: src/renderer/components/GeneralSettings/styles.tsx [hunk 1]
```diff
@@ -71,4 +71,19 @@ export const InfoButton = styled.h6`
...
+  border: 2px solid;
+  border-color: #fa3516;
+  border-radius: 5px;
+  cursor: pointer;
+
+  :hover {
+    color: ${colors.mutedTextDark} !important;
+  }
+`;
+
```
Comment (nistei) [inline src/renderer/components/GeneralSettings/styles.tsx:88]: Could probably make it a button now instead of h6

**Review by nistei — CHANGES_REQUESTED** [review]

File: src/renderer/components/GeneralSettings/styles.tsx [hunk 2]
```diff
@@ -71,4 +71,19 @@ export const InfoButton = styled.h6`
   }
 `;
 
+export const ResetButton = styled.h6`
+  padding: 0.2em 0.5em 0.2em 0.5em;
+  margin-top: 1.5em;
+  color: white !important;
+  background: #fa3516;
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/GeneralSettings/styles.tsx:78]: `background: #fc3a3a;`
different red than the one suggested by Harry

File: src/renderer/components/GeneralSettings/styles.tsx [hunk 3]
```diff
@@ -71,4 +71,19 @@ export const InfoButton = styled.h6`
...
+  border: 2px solid;
+  border-color: #fa3516;
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/GeneralSettings/styles.tsx:80]: `border-color: #fc3a3a;`
different red than the one suggested by Harry

File: src/renderer/components/GeneralSettings/styles.tsx [hunk 4]
```diff
@@ -71,4 +71,19 @@ export const InfoButton = styled.h6`
...
+  border-radius: 5px;
+  cursor: pointer;
+
+  :hover {
+    color: ${colors.mutedTextDark} !important;
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/GeneralSettings/styles.tsx:85]: the text remains white on the similar buttons while hovering

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]

**Review by nistei — APPROVED** [review]

**Pull request merged**